tox.ini

```diff
@@ -32,6 +34,9 @@ deps =
     devdeps: pyerfa>=0.0.dev0
     devdeps: git+https://github.com/astropy/pyvo.git#egg=pyvo
 
+    # astropy doesn't yet have a 3.13 compatible release
+    py313: astropy>0.0dev0
```

Comment:
Missing equal sign?
```suggestion
    py313: astropy>=0.0dev0
```

Reply:
Might also need pyerfa-dev (not sure).

Reply:
Nope, it works well just as is and I would rather avoid pulling in more than necessary dev versions on this job. Once an astropy release is out on pypi with a 3.13 support I'll move to clean this up, too.